invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentMetadata/DocumentMetadata.js

```diff
@@ -217,27 +234,75 @@ export default class DocumentMetadata extends Component {
     }
   }
 
+  setRestrictions = selections => {
+    const selectedIds = selections.map(selection => selection.title);
```

Comment:
I think here is the moment to ensure that the emails should be forced to lowercase :) @kprzerwa @ntarocco WDYT?

Reply:
it should be normalized before save, no? It would ensure that wherever we set the restriction by the endpoint, it will be handled